packages/block-library/src/index.native.js

```diff
@@ -118,6 +118,10 @@ const registerBlock = ( block ) => {
 	} );
 };
 
+// only enable code block for development
+// eslint-disable-next-line no-undef
+const devOnly = ( block ) => !! __DEV__ ? block : null;
```

Comment:
These changes came over after rebasing. Not sure why they're reading as a change here though even though they exist in  master already https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/index.native.js#L121-L123

Reply:
It looks like an older revision of master was merged that didn't contain that change:

![Screen Shot 2020-01-14 at 08 54 40](https://user-images.githubusercontent.com/8739/72324604-9bd5a980-36ab-11ea-8fd9-4e977bac190a.png)

Reply:
Hmm, I just rebased again and off of master and it still shows up. I did see that there is a bug in Github related to extra commits in a PR after a rebase https://stackoverflow.com/questions/16306012/github-pull-request-showing-commits-that-are-already-in-target-branch 

I took care of other ones by switching branches but that doesn't seem to be working here.  I'll keep looking.

Reply:
There was an extra commit "Apply changes from master ..." where I had merged when I should have rebased. So I reverted that commit and rebased again. We're good now.